### ISSUE TITLE
add support for client certs to get_url

### ISSUE THREAD

**Issue by vegaed** [ansible/ansible-modules-core#617]
##### ISSUE TYPE

Feature Idea
##### COMPONENT NAME

get_url module
##### ANSIBLE VERSION

N/A
##### SUMMARY

All webpages in our environment are https and require a client certificate to authenticate. It would be useful to have an option to pass along a client cert.

**Comment by ansibot** [comment]
@jpmens ping, this issue is waiting for your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@jpmens, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by jpmens** [comment]
needs_contributor

**Comment by jaredledvina** [comment]
It'd be awesome if this wasn't only get_url, but also the uri module. (Maybe that warrants a separate issue though)

**Comment by ansibot** [comment]
This repository has been locked. All new issues and pull requests should be filed in https://github.com/ansible/ansible

Please read through the <a href="http://docs.ansible.com/ansible/dev_guide/repomerge.html#move-issues-and-prs-to-new-repo">repomerge</a> page in the dev guide. The guide contains links to tools which automatically move your issue or pull request to the ansible/ansible repo.

**Comment by ansibot** [comment]
This issue was migrated to https://github.com/ansible/ansible/issues/30186
